services/api.js

```diff
@@ -45,7 +45,9 @@ const api: APIMethods = METHODS.reduce(
         .post(`http://${RPC.host}:${RPC.port}`, {
           method: 'POST',
           json: true,
-          auth: `${RPC.user}:${RPC.password}`,
+          headers: {
```

Comment:
Out of curiosity. Why have changed from auth to headers + Authorization?

Reply:
this allows the fallback to use the standard zcash RPC (with cookies file) authentication and also the current authentication that we are already using